packages/paste-website/src/pages/components/list/index.mdx

```diff
@@ -0,0 +1,237 @@
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/list/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/list"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/list/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/list"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-list--ordered-list"
+  abstractUrl="https://share.goabstract.com/85c81258-90b9-49c8-b34b-9418da5f2f00"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<LivePreview scope={{UnorderedList, ListItem}} language="jsx">
+  {`<UnorderedList>
+  <ListItem>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus.
+  </ListItem>
+  <ListItem>
+    Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </ListItem>
+</UnorderedList>`}
+</LivePreview>
+
+## Guidelines
+
+Lists can be ordered or unordered. If the order of the list items matters for context to the user, an ordered list should be used. If the meaning of the list is not changed based on the order of the items, an unordered list should be used.
+
+### About Lists
+
+Lists are a number of connected list items that are printed consecutively, typically one after the other. Lists can be ordered, using numbers as an indicator to the reader, or unordered with bullet points.
+
+- Ordered lists are displayed with a preceding number
+- Unordered lists are displayed with a preceding bullet point
+- Lists can be nested as deeply as needed, and they may alternate between ordered and unordered
+- Lists should align with the font-size, font-weight, and line-height of paragraphs, but are styled with bullets or numbers
+- Lists can have similar children elements as paragraphs to provide emphasis on a certain word or phrase
+
+List is part of the [Typography package](/components/typography), which includes Header and Paragraph as well.
+
+### Accessibility
+
+- List text should meet AA requirements (4.5:1) for color contrast from itself and the background color
+- `ListItem`s should always have an `OrderedList` or `UnorderedList` parent component
+- Nested `ListItem`s should be wrapped in their own, new `OrderedList` or `UnorderedList` parent component
```

Comment:
This is a little hard to understand.  I think an example (DO/DONT) here would be appreciated by folks.

Reply:
updated